src/app/profile/settings/notifications/notifications.component.html

```diff
@@ -0,0 +1,91 @@
+<div class="container-fluid container-cards-pf">
+  <div class="row-cards-pf">
+    <div class="col-xs-10 col-sm-6 col-md-6">
+      <div class="card-pf">
+        <div class="card-pf-heading">
+          <div class="row notification-option-content">
+            <div class="col-xs-6 col-sm-6 col-md-6">
+              <h2 class="card-pf-title">Delivery Methods</h2>
+            </div>
+          </div>
+        </div>
+        <div class="card-pf-body">
+          <div class="row github-notification-option-info">
+            <div class="col-xs-1 col-sm-1 col-md-1">
+              <input type="checkbox" [disabled]="true" [checked]="true">
+            </div>
+            <div class="notification-option-text-box col-xs-10 col-sm-10 col-md-10">
+              <b>System Widget Notifications</b>
+              <br>
+              <p>These will only be presented in the notification drawer, represented by the [] icon.</p>
+            </div>
+          </div>
+          <div class="row github-notification-option-info">
+            <div class="col-xs-1 col-sm-1 col-md-1">
+                <input type="checkbox" [disabled]="true" [checked]="true">
+              </div>
+            <div class="notification-option-text-box col-xs-10 col-sm-10 col-md-10">
+              <b>Email Notifications</b>
+              <br>
```

Comment:
It is preferred to not use `<br>` and apply a margin to the bottom of "Email Notifications".